assets/js/admin.js

```diff
@@ -528,3 +528,18 @@
 		});
 	});
 })(this);
+
+//* The below is run after the DOM has fully loaded */
+(function (window, undefined) {
+	'use strict';
+
+	function onLoadHandler() {
+		var currentUrl = window.location.href;
+		if (currentUrl.includes('code=')) {
+			/* Remove all parameters after 'code' */
+			var newUrl = currentUrl.replace(/([?&])code=[^&]+(.*)$/, '$1');
+			window.history.replaceState({}, document.title, newUrl);
```

Comment:
Why is this needed? Also, shouldn't this be part of pro rather than free?

Reply:
After returning from google  we updated the code in the option table and  and then removed code and other param in url  will be remove by this jquery code.

there is no js file in pro addon. and all the js codes are written in free plugin so I have added this one here.

Reply:
Why is there a need to remove the code from URL?

> there is no js file in pro addon. and all the js codes are written in free plugin so I have added this one here.
That means we need to create one. No harm in doing that.

Reply:
if the user reload the page then it will update the code every time and I think this is not a good practice and there are other param for which I think people may be confused so I removed that.
 we can left it as it is if you're ok with this

Reply:
If we dont want to remove that from url then we don't need to create new js file in pro addon.

Reply:
Yeah, no need to remove